tests/sql/setup_test_docx_etl.sql

```diff
@@ -0,0 +1,18 @@
+insert into "Bills" (legislative_id, legislative_session, bill_number, session_type, chamber)
+values(
+   '20230HB1540',
+   '2023',
+   '1540',
+   '0',
+   'HOUSE'
+)
+
+insert into "Revisions" (revision_guid, printer_no, full_text_link, publication_date, description, bill_internal_id)
```

Comment:
consider making this idempotent, using something like `insert ignore` command. As it stands, if in any situation the .sql query is run more than once, an error will be thrown

Reply:
See my above comment re: insert ignore